actiontext/test/template/form_helper_test.rb

```diff
@@ -61,7 +81,42 @@ class ActionText::FormHelperTest < ActionView::TestCase
       output_buffer
   end
 
-  def form_with(*)
-    @output_buffer = super
+  test "form with rich text area having placeholder without locale" do
+    form_with model: Message.new, scope: :message do |form|
+      form.rich_text_area :content, placeholder: true
+    end
+
+    assert_dom_equal \
+      '<form action="/messages" accept-charset="UTF-8" data-remote="true" method="post">' \
+        '<input type="hidden" name="message[content]" id="message_content_trix_input_message" />' \
+        '<trix-editor placeholder="Content" id="message_content" input="message_content_trix_input_message" class="trix-content" data-direct-upload-url="http://test.host/rails/active_storage/direct_uploads" data-blob-url-template="http://test.host/rails/active_storage/blobs/:signed_id/:filename">' \
+        "</trix-editor>" \
+      "</form>",
+      output_buffer
   end
+
+  test "form with rich text area having placeholder with locale" do
+    with_locale :placeholder do
+      form_with model: Message.new, scope: :message do |form|
+        form.rich_text_area :title, placeholder: true
+      end
+    end
+
+    assert_dom_equal \
+      '<form action="/messages" accept-charset="UTF-8" data-remote="true" method="post">' \
+        '<input type="hidden" name="message[title]" id="message_title_trix_input_message" />' \
+        '<trix-editor placeholder="Story title" id="message_title" input="message_title_trix_input_message" class="trix-content" data-direct-upload-url="http://test.host/rails/active_storage/direct_uploads" data-blob-url-template="http://test.host/rails/active_storage/blobs/:signed_id/:filename">' \
+        "</trix-editor>" \
+      "</form>",
+      output_buffer
+  end
+
+  private
+
+    def with_locale(testing_locale = :label)
+      old_locale, I18n.locale = I18n.locale, testing_locale
+      yield
+    ensure
+      I18n.locale = old_locale
+    end
```

Comment:
Can we use `I18n.with_locale` instead?

```ruby
test "form with rich text area having placeholder with locale" do
  I18n.with_locale :placeholder do
    form_with model: Message.new, scope: :message do |form|
      form.rich_text_area :title, placeholder: true
    end
  end

  # ...
```

Reply:
@georgeclaghorn : Yep. Used `I18n.with_locale` instead of `with_locale`.